tests/types/test_private_field.py

```diff
@@ -0,0 +1,61 @@
+import strawberry
+
+
+def test_private_field():
+    @strawberry.type
+    class Query:
+        name: str
+        age: strawberry.Private[int]
+
+    definition = Query._type_definition
+
+    assert definition.name == "Query"
+    assert len(definition.fields) == 1
+
+    assert definition.fields[0].name == "name"
+    assert definition.fields[0].type == str
+
+    instance = Query(name="Luke", age=22)
+    assert instance.name == "Luke"
+    assert instance.age == 22
+
+
+def test_private_field_with_strawberry_field():
+    @strawberry.type
+    class Query:
+        name: str
+        age: strawberry.Private[int] = strawberry.field(description="🤫")
```

Comment:
what's the use case for a private field being strawberry field?

Reply:
I guess there isn't one, I just wanted to check if it was possible. It might make more sense to raise an error if you try and do this. What do you think @AndrewIngram ?

Reply:
It feels like using strawberry.field contradicts the reason behind private fields, so raising an error might make sense.

Reply:
I agreed with raising an error 😊

Reply:
Done